resolwe/flow/models/data.py

```diff
@@ -490,6 +496,8 @@ def duplicate(self, contributor=None):
         duplicate = Data.objects.get(id=self.id)
         duplicate.pk = None
         duplicate.slug = None
+        duplicate.entity = None
```

Comment:
Why don't we inherit `entity` and `collection` from the original data object? Consider supporting `inherit_entity=False`, `inherit_collection=False`.